src/ResourceManager/RecoveryServices.Backup/RecoveryServices.Backup/ProviderModel/IPsBackupProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.ProviderModel
```

Comment:
cmdlets is separate layer right. do you want to put providerModel under cmdlets namespace?